libcaf_core/src/detail/rfc3629.cpp

```diff
@@ -42,9 +63,6 @@ constexpr bool is_continuation_byte(std::byte value) noexcept {
 bool validate_rfc3629(const std::byte* first, const std::byte* last) {
   while (first != last) {
     auto x = *first++;
-    // Null byte (terminator) is not allowed.
-    if (x == 0_b)
-      return false;
```

Comment:
Why did you drop this?

Reply:
`0` is the first UTF code point, and is valid, at least according to Wikipedia and autobahn. 
There is an autobahn test that sends a payload of length 1 containing `0x00` so it made sense to remove it.

Reply:
Well then. 🤷🏻‍♂️